### ISSUE TITLE
Does this work with RDS?

### ISSUE THREAD

**Issue by AdityaAnand1** [toluaina/pg-sync#21]
There's no way to set wal_level parameter for RDS DBs 😕

**Comment by sdreger** [comment]
There is a way:

To [enable logical decoding](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html) for an Amazon RDS for PostgreSQL DB instance:
- The user account requires the **rds_superuser** role to enable logical replication. The user account also requires the **rds_replication** role to grant permissions to manage logical slots and to stream data using logical slots.
- Set the `rds.logical_replication` static parameter to 1. As part of applying this parameter, we also set the parameters `wal_level`, `max_wal_senders`, `max_replication_slots`, and `max_connections`. These parameter changes can increase WAL generation, so you should only set the `rds.logical_replication` parameter when you are using logical slots.
- Reboot the DB instance for the static `rds.logical_replication` parameter to take effect.
- Create a logical replication slot as explained in the next section. This process requires that you specify a decoding plugin. Currently we support the test_decoding output plugin that ships with PostgreSQL.

But it doesn't help because of difference in RDS permissions model. See: https://github.com/toluaina/pg-sync/issues/16

**Comment by AdityaAnand1** [comment]
Ahh, got it. But it still doesn't work?

Did you find any alternative solutions or workarounds for syncing postgres data to elasticsearch?

**Comment by toluaina** [comment]
Yes, I am still exploring how to do this. This needs to be implemented so its RDS agnostic.

**Comment by toluaina** [comment]
Duplicate of the previous issue,. Should be working with RDS now.

**Comment by ashwinj92** [comment]
Hello, I'm aware this is an old issue and I'm trying to use PGSync with RDS, does it work without changing `wal_level= logical` and `max_replication_slots = 1` ?
